# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/TabChild.cpp
# Commit: 102881458323
# Full Hash: 1028814583232487b52b9c20d47e3b38dc1c288a
# Author: Barret Rennie <barret@brennie.ca>
# Date: 2019-04-04 04:33:36
# Regressor Bug: 1510569
# File Overlap Count: 1
# Description:
#   Bug 1510569 - Implement nsIWebProgressListener2 for TabChild r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D25648
# ==============================================================================

diff -r d5106af44dab -r 102881458323 dom/ipc/TabChild.cpp
--- a/dom/ipc/TabChild.cpp	Wed Apr 03 17:32:14 2019 +0000
+++ b/dom/ipc/TabChild.cpp	Wed Apr 03 17:32:21 2019 +0000
@@ -3342,6 +3342,21 @@
   return NS_OK;
 }
 
+NS_IMETHODIMP TabChild::OnProgressChange64(nsIWebProgress* aWebProgress,
+                                           nsIRequest* aRequest,
+                                           int64_t aCurSelfProgress,
+                                           int64_t aMaxSelfProgress,
+                                           int64_t aCurTotalProgress,
+                                           int64_t aMaxTotalProgress) {
+  return NS_ERROR_NOT_IMPLEMENTED;
+}
+
+NS_IMETHODIMP TabChild::OnRefreshAttempted(nsIWebProgress* aWebProgress,
+                                           nsIURI* aRefreshURI, int32_t aMillis,
+                                           bool aSameURI, bool* aOut) {
+  return NS_ERROR_NOT_IMPLEMENTED;
+}
+
 nsresult TabChild::PrepareProgressListenerData(
     nsIWebProgress* aWebProgress, nsIRequest* aRequest,
     Maybe<WebProgressData>& aWebProgressData, RequestData& aRequestData) {